### ISSUE TITLE
2.0.1 broken

### ISSUE THREAD

**Issue by jluttine** [tk0miya/testing.common.database#20]
I have used 2.0.0 for some time and upgrading to 2.0.1 broke all my db-based unit tests. The tests fail almost immediately with the following kind of error for each test:

```
    def wait_booting(self):
        boot_timeout = self.settings.get('boot_timeout', self.DEFAULT_BOOT_TIMEOUT)
        exec_at = datetime.now()
        while True:
            if self.child_process.poll() is not None:
                raise RuntimeError("*** failed to launch %s ***\n" % self.name +
>                                  self.read_bootlog())
E               RuntimeError: *** failed to launch Postgresql ***
E               FATAL:  lock file "postmaster.pid" already exists
E               HINT:  Is another postmaster (PID 1933) running in data directory "/tmp/tmpv9dv74gc/data"?
```

I wasn't able to dig deeper into it. Do you need some more information and how do I get it?

**Comment by vikt0rs** [comment]
Seems to be duplicate of https://github.com/tk0miya/testing.common.database/issues/16

**Comment by tk0miya** [comment]
Fixed by #18. I will release this ASAP.
